stdlib/functools.pyi

```diff
@@ -99,9 +99,12 @@ def total_ordering(cls: type[_T]) -> type[_T]: ...
 def cmp_to_key(mycmp: Callable[[_T, _T], int]) -> Callable[[_T], SupportsAllComparisons]: ...
 
 class partial(Generic[_T]):
-    func: Callable[..., _T]
-    args: tuple[Any, ...]
-    keywords: dict[str, Any]
+    @property
```

Comment:
Same here, these are read-write in Python but readonly in C `_functools.partial`. We should follow the C version as that's what people will be using virtually all of the time.

Reply:
Also: if it's only read-only in very specific circumstances, it arguably won't be type-safe for end users to try to write to that attribute in any circumstances